app/views/users/show.html.erb

```diff
@@ -32,6 +32,15 @@
     </ul>
   </div>
```

Comment:
- [ ] **[OPTIONAL]** The posts on this page should be clickable links that lead to the respective posts.